### DISCUSSION TITLE
conversion directly to tracker formats such as .it

### DISCUSSION THREAD

**Discussion by H-A-M-G-E-R** [vgmtrans/vgmtrans#349] [General]
vgm -> convert to .it -> use mITroid to convert to a format which can be inserted into Super Metroid

**Comment by MugUK** [comment]
Years ago I wrote a guide on converting the MID & DLS files into Impulse Tracker. 

My process was as follows:

1) Ripping the MIDI and Soundfonts out of the ROM using NDS Explorer 2. 
2) Then importing them into Awave Studio 10.2. 
3) Exporting them into RMI files, which were then imported into Open MOD Plug Tracker. 
4) Finally OMPT exported them into IT files - after removing all unused tracks and samples to make the files a bit smaller.

Full guide -> http://mug-uk.co.uk/nintendo-ds/

Is there now a quicker way of doing this?

**Comment by H-A-M-G-E-R** [comment]
snes spc has 8 tracks
want to convert to .it so that it has 8 tracks
